### DISCUSSION TITLE
EME Frontend Submit

### DISCUSSION THREAD

**Discussion by teamtacho** [liedekef/events-made-easy#619] [General]
Hi Team
there is a new embedded functionallity for the frontend submit page. Please can you provide any template or eamples how to create a form. The old page from the previous plugin cannot transferred easily.

**Comment by liedekef** [comment]
Look in the settings of EME, tab "frontend submit". There a template is provided (almost identical to the old plugin)

**Comment by teamtacho** [comment]
Hi Franky

Unfortunately, I had deleted the content in the tab field.

Can you tell me where to find it or send it to me, please.

 

Best Regards Michael

 

Von: Franky Van Liedekerke ***@***.***> 
Gesendet: Mittwoch, 31. Juli 2024 00:01
An: liedekef/events-made-easy ***@***.***>
Cc: teamtacho ***@***.***>; Author ***@***.***>
Betreff: Re: [liedekef/events-made-easy] EME Frontend Submit (Discussion #619)

 

Look in the settings of EME, tab "frontend submit". There a template is provided (almost identical to the old plugin)

**Comment by liedekef** [comment]
This is the default:
```
<h2>Event Information</h2>
		<div class="input">
			<label for="event_name">Event Name</label><br />
			#_FIELD{event_name}
		</div>
                <div class="input select">
                        <label for="event_category_ids">Select the Event Category</label><br/>
			#_FIELD{event_category_ids}
                </div>
		<fieldset>
			<legend>Date & Time</legend>
			<fieldset id="event_start">
				<legend>Start</legend>
				<div class="input">
					<label for="event_start_date">Date</label>
					#_FIELD{event_start_date}
				</div>
				<div class="input">
					<label for="event_start_time">Time</label> 
					#_FIELD{event_start_time}
				</div>
			</fieldset>
			
			<fieldset id="event_end">
				<legend>End</legend>
				<div class="input">
					<label for="event_end_date">Date</label>
					#_FIELD{event_end_date}
				</div>
				<div class="input">
					<label for="event_end_time">Time</label> 
					#_FIELD{event_end_time}
				</div>
			</fieldset>
		</fieldset>
                <div class="input">
                        <label for="event_description">Description</label><br />
			#REQ_FIELD{event_notes}
                </div>

                <div class="input">
                        <label for="event_contactperson_email_body">Contact E-mail body</label><br />
			#_FIELD{event_contactperson_email_body}{textarea}
                </div>

                <div class="input">
                        <label for="event_url">Event External URL</label><br />
			#_FIELD{event_url}
                </div>
                <h3>Location Information</h3>
                <div class="input">
                        <label for="location_name">Location name</label>
                        #_FIELD{location_name}
                </div>

                <div class="input">
                        <label for="location_address1">Address</label>
                        #_FIELD{location_address1}
                </div>

                <div class="input">
                        <label for="location_city">City or Town</label>
                        #_FIELD{location_city}
                </div>

                <div class="map">
                        #_MAP
                </div>
'
```

**Comment by teamtacho** [comment]
Thanks a lot!

 

But there are no shortcodes possible.

Is there a way to use a template?

 

Von: Franky Van Liedekerke ***@***.***> 
Gesendet: Mittwoch, 31. Juli 2024 08:45
An: liedekef/events-made-easy ***@***.***>
Cc: teamtacho ***@***.***>; Author ***@***.***>
Betreff: Re: [liedekef/events-made-easy] EME Frontend Submit (Discussion #619)

 

This is the default:

<h2>Event Information</h2>
               <div class="input">
                       <label for="event_name">Event Name</label><br />
                       #_FIELD{event_name}
               </div>
                <div class="input select">
                        <label for="event_category_ids">Select the Event Category</label><br/>
                       #_FIELD{event_category_ids}
                </div>
               <fieldset>
                       <legend>Date & Time</legend>
                       <fieldset id="event_start">
                               <legend>Start</legend>
                               <div class="input">
                                       <label for="event_start_date">Date</label>
                                       #_FIELD{event_start_date}
                               </div>
                               <div class="input">
                                       <label for="event_start_time">Time</label> 
                                       #_FIELD{event_start_time}
                               </div>
                       </fieldset>
                       
                       <fieldset id="event_end">
                               <legend>End</legend>
                               <div class="input">
                                       <label for="event_end_date">Date</label>
                                       #_FIELD{event_end_date}
                               </div>
                               <div class="input">
                                       <label for="event_end_time">Time</label> 
                                       #_FIELD{event_end_time}
                               </div>
                       </fieldset>
               </fieldset>
                <div class="input">
                        <label for="event_description">Description</label><br />
                       #REQ_FIELD{event_notes}
                </div>
 
                <div class="input">
                        <label for="event_contactperson_email_body">Contact E-mail body</label><br />
                #_FIELD{event_contactperson_email_body}{textarea}
                </div>
 
                <div class="input">
                        <label for="event_url">Event External URL</label><br />
                       #_FIELD{event_url}
                </div>
                <h3>Location Information</h3>
                <div class="input">
                        <label for="location_name">Location name</label>
                        #_FIELD{location_name}
                </div>
 
                <div class="input">
                        <label for="location_address1">Address</label>
                        #_FIELD{location_address1}
                </div>
 
                <div class="input">
                        <label for="location_city">City or Town</label>
                        #_FIELD{location_city}
                </div>
 
                <div class="map">
                        #_MAP
                </div>
'

**Comment by liedekef** [comment]
The next version of EME will allow generic placeholders and shortcodes in the FS form.
And yes, templating is possible, see https://www.e-dynamics.be/wordpress/frontend-submit/

**Reply by liedekef** [reply]
Concernng the generic placeholders and shortcode 'fix': see https://github.com/liedekef/events-made-easy/commit/af410ff3c62bb0cbf261fd70f0a0e99d188ed7dc
